packages/desktop-client/src/components/accounts/Account.tsx

```diff
@@ -944,7 +947,7 @@ class AccountInternal extends PureComponent<
   onCreatePayee = (name: string) => {
     const trimmed = name.trim();
     if (trimmed !== '') {
-      return this.props.dispatch(createPayee(name));
+      return this.props.dispatch(createPayee({ name })).unwrap();
```

Comment:
Not a huge fan of this `.unwrap` API tbh, wish it threw by default like a normal Promise :/